docs/src/modules/java/pages/component-and-service-calls.adoc

```diff
@@ -75,31 +75,28 @@ include::java:example$doc-snippets/src/main/java/com/example/callanotherservice/
 === Over gRPC
 The service is identified by the name it has been deployed. Akka takes care of routing requests to the service and keeping the data safe by encrypting the connection and handling authentication for you.
 
-ifdef::todo[TODO: a more realistic sample here, possibly aligned with the HTTP sample above]
-ifdef::todo[TODO: cover streaming calls with an example?]
-
-In this sample we will make an action that does a call to the xref:grpc-endpoints.adoc[gRPC endpoints example] service, deployed with the service name `doc-snippets`.
+In this sample we will make an endpoint that does a call to the xref:grpc-endpoints.adoc[gRPC endpoints] customer registry service, deployed with the service name `customer-registry`.
 
 The SDK provides `akka.javasdk.grpc.GrpcClientProvider` which provides gRPC client instances for calling other services.
```

Comment:
Should we list from where that is possible/makes sense already here? (workflows, consumers, HTTP and gRPD endpoints, timers)

Reply:
I think it works fine at the bottom... made it a `NOTE:` now so it gets a bit more relevance.. but I think it's not essential to put here because even if they try, it will fail locally anyway with a proper message.. so it's not something they will only hit later in prod, etc. But let me know what you think.. I mean, I also don't mind moving it here if you prefer..